libs/akita/src/lib/store.ts

```diff
@@ -4,17 +4,23 @@ import { resetCustomAction, setAction } from './actions';
 import { getAkitaConfig, getGlobalProducerFn } from './config';
 import { deepFreeze } from './deepFreeze';
 import { dispatchAdded, dispatchDeleted, dispatchUpdate } from './dispatchers';
-import { __DEV__, isDev } from './env';
+import { isDev } from './env';
 import { assertStoreHasName } from './errors';
 import { isDefined } from './isDefined';
 import { isFunction } from './isFunction';
 import { isPlainObject } from './isPlainObject';
 import { isBrowser } from './root';
 import { configKey, StoreConfigOptions, UpdatableStoreConfigOptions } from './storeConfig';
-import { __stores__ } from './stores';
 import { commit, isTransactionInProcess } from './transaction';
 import { StoreCache, UpdateStateCallback } from './types';
 
+/** @internal */
+export const __stores__: { [storeName: string]: Store<any> } = {};
```

Comment:
Wht it's here?

Reply:
I had to move it here to fix a cyclic import